### PR TITLE
Add an example for PHP 8

### PR DESCRIPTION
**Pull request by akerouanton**

### CODE REVIEW

File: examples/angular-realworld-example-app/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ To build and run this example you need:
 ##### Run this example
 
 ```bash
-$ cd examples/php/angular-realworld-example-app
+$ cd examples/angular-realworld-example-app
```
Comment (nm2107) [inline examples/angular-realworld-example-app/README.md:16]: :+1:

File: examples/symfony-demo/zbuild.yml [hunk 2]
```diff
@@ -21,10 +21,18 @@ sources:
...
+  - data/
 
 integrations:
   - blackfire
 
+post_install:
+  - echo '' > .env
+  # @TODO: let zbuild create data/ folder with the right owner/group
+  - mkdir -p var/cache/ data/
+  # @TODO: find a proper way to warmup symfony cache
```
Comment (nm2107) [inline examples/symfony-demo/zbuild.yml:33]: use composer scripts ?

Comment (NiR-) [reply examples/symfony-demo/zbuild.yml:33]: They won't work for this example because Doctrine tries to connect to the DB to know its version :confused:

Comment (nm2107) [reply examples/symfony-demo/zbuild.yml:33]: rah still this issue :(

**Pull request merged**